src/ImageProvider/ImageProviderConfig.py

```diff
@@ -1,9 +1,21 @@
 class ImageProviderConfig:
-    def __init__(self, azure_blob_key: str = "key not set", azure_blob_account: str = "swisstopo", azure_blob_name: str = "rawdata", image_url: str = "../data/in/ortho"):
+    def __init__(self, 
+            azure_blob_name: str = None , 
```

Comment:
I don't quite understand how this is better than having the values as actual defaults of the parameters 🤔